pkg/store/sqlite/search.go

```diff
@@ -14,6 +17,7 @@ import (
 
 const (
 	counterpartySearchSQL       = "SELECT id, name FROM counterparties ORDER BY name ASC"
+	counterpartyDomainSearchSQL = "SELECT id, website FROM counterparties WHERE website LIKE :domainParam UNION select id, endpoint FROM counterparties WHERE endpoint LIKE :domainParam"
```

Comment:
Basically I'm normalizing the URL to a domain; e.g. `https://example.com` ⟶ `%example.com%` and then I'm doing a LIKE. I then rank the search results using a fuzzy distance score.